lsp-flycheck.el

```diff
@@ -58,19 +58,19 @@ CALLBACK is the status callback passed by Flycheck."
   "Setup Flycheck for use with lsp-mode."
   ;; Disable automatic syntax checks, since lsp-mode will call `flycheck-buffer'
   ;; directly.
-  (add-to-list 'flycheck-checkers 'rust-lsp)
   (setq-local flycheck-check-syntax-automatically nil
               flycheck-checker 'rust-lsp)
```

Comment:
Or rather, it needs to be the same as the checker defined below. But that checker should be namespaced properly, so probably `lsp-flycheck-checker`.